server/controllers/temporary-accommodation/manage/assessmentsController.ts

```diff
@@ -0,0 +1,62 @@
+import type { Request, RequestHandler, Response } from 'express'
+import AssessmentsService from '../../../services/assessmentsService'
+import extractCallConfig from '../../../utils/restUtils'
+
+export const assessmentsTableHeaders = [
```

Comment:
Ah, yeah, had a quick try myself at putting this in the njk file and it seems non-obvious how to make it work. Suggest update the commit message to be more certain about why this is here